src/engraving/libmscore/textbase.cpp

```diff
@@ -968,11 +988,11 @@ mu::draw::Font TextFragment::font(const TextBase* t) const
         }
     } else {
         family = format.fontFamily();
+        font.setBold(format.bold());
+        font.setItalic(format.italic());
     }
 
     font.setFamily(family);
-    font.setBold(format.bold());
-    font.setItalic(format.italic());
     Q_ASSERT(m > 0.0);
```

Comment:
Much simpler and more reliable to ignore bold/italic flags here than when setting the CharFormat